IBPSA/Airflow/Multizone/BaseClasses/PowerLawResistance.mo

```diff
@@ -96,119 +30,30 @@ equation
     d=d,
     dp_turbulent=dp_turbulent);
 
-  port_a.m_flow = rho*V_flow;
-
-  // Isenthalpic state transformation (no storage and no loss of energy)
-  port_a.h_outflow = inStream(port_b.h_outflow);
-  port_b.h_outflow = inStream(port_a.h_outflow);
-
-  // Mass balance (no storage)
-  port_a.m_flow + port_b.m_flow = 0;
-
-  // Transport of substances
-  port_a.Xi_outflow = inStream(port_b.Xi_outflow);
-  port_b.Xi_outflow = inStream(port_a.Xi_outflow);
 
-  port_a.C_outflow = inStream(port_b.C_outflow);
-  port_b.C_outflow = inStream(port_a.C_outflow);
   annotation (
     Documentation(info="<html>
-<p>
-This model describes the mass flow rate and pressure difference relation
-of an orifice in the form
-</p>
-<pre>
-    V_flow = k * dp^m,
-</pre>
-<p>
-where <code>k</code> is a variable and
-<code>m</code> a parameter.
-For turbulent flow, set <code>m=1/2</code> and
-for laminar flow, set <code>m=1</code>.
-</p>
-<p>
-The model is used as a base for the interzonal air flow models.
-</p>
+<p>This model describes the mass flow rate and pressure difference relation in the form </p>
+<p><span style=\"font-family: Courier New;\">V_flow = k * dp^m,</span></p>
+<p>where <span style=\"font-family: Courier New;\">k</span>  and <span style=\"font-family: Courier New;\">m</span> are parameters. For turbulent flow, set <span style=\"font-family: Courier New;\">m=1/2</span> and for laminar flow, set <span style=\"font-family: Courier New;\">m=1</span>. </p>
```

Comment:
@kldjonge whenever you make changes to the html documentation and revision history using the dymola 'info editor', dymola reformats the html code, which leads to very long lines as you can see here. We try to format everything as in the original format to ensure readability. Therefore please revert these changes and use the dymola 'info source' tab to edit the documentation instead of the 'info editor'.